### ISSUE TITLE
[Programmatic Access – Make Code – Create a project] Ensures every form element has a label (select)

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3557]
**User Experience:**
If instructions or labels does not clearly identify the controls in a form, the users will not know what input data is expected. The pages will look cluttered with unnecessary information and   people with disabilities will miss important cues and instructions

**Note**: User credentials should NOT be included in the bug.

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate to the controls appearing under "My Project" section in Home page and select 'New Project' Button.
4. Navigate all controls under 'Create a Project' pop-up.
5. Run accessibility insights for web tool and select fast pass.
6. Verify the issue.

**Actual Result:**
Label is not properly defined for 'Code option combo-box' under 'Create a project' pop-up.

**Expected Result:**
Label should be properly define for 'Code option combo-box' under 'Create a project' pop-up.

**Issue:** Ensures every form element has a labe

**Target application:** Microsoft MakeCode for micro:bit

**Element path:** select

**Snippet**: <select class="ui dropdown"><option aria-selected="true" value="">Blocks, JavaScript, and Python</option><option aria-selected="false" value="python-only">Python Only</option><option aria-selected="false" value="javascript-only">JavaScript Only</option></select>

**How to fix:** 
Fix any of the following:
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Form element does not have an implicit (wrapped) <label>
  Form element does not have an explicit <label>
  Element has no title attribute or the title attribute is empty
  Element's default semantics were not overridden with role="none"
  Element's default semantics were not overridden with role="presentation"

This accessibility issue was found using Accessibility Insights for Web 2.22.0 (axe-core 4.0.2), a tool that helps find and fix accessibility issues. Get more information & download this tool.

**Note:**
Same issue exists on Firefox & Chrome.

**WCAG Reference:**
[WCAG 1.3.1 - Info and Relationships
](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Tool: Accessibility insights for web
Screen Reader: Narrator, NVDA, JAWS

**Comment by Shalini3457** [comment]
Labels:
#Accessibility, #A11ySev2, #HCL, #Benchmark-HCL-MakeCode-OCT2020, #WCAG1.3.1, #Win10-Edge(Chromium)

![WCAG1 3 1_Create a project_Ensures every form element has a label](https://user-images.githubusercontent.com/72871089/97955927-67afae80-1dcd-11eb-8478-f67fe3b178d5.jpg)
